build/deploy/mkimage.sh

```diff
@@ -12,26 +12,35 @@
 
 set -euo pipefail
 
-cd -P "$(dirname $0)"
-DIR=$(pwd -P)
+# This is mildly trick: This script runs itself recursively. The first
+# time it is run it does not take the if-branch below and executes on
+# the host computer. It uses the builder.sh script to run itself
+# inside of docker passing "docker" as the argument causing the
+# commands in the if-branch to be executed within the docker
+# container.
+if [ "${1:-}" = "docker" ]; then
+    time make testbuild
```

Comment:
Is building the tests statically like this worthwhile? It takes quite a while and seems duplicative of the tests that are run build `circle-test.sh`.